crates/bevy_render/src/camera/bundle.rs

```diff
@@ -75,6 +75,23 @@ pub struct OrthographicCameraBundle {
 }
 
 impl OrthographicCameraBundle {
+    /// Create an orthographic projection camera to render 2D content.
+    ///
+    /// The projection creates a camera space where X points to the right of the screen,
+    /// Y points to the top of the screen, and Z points out of the screen (backward),
+    /// forming a right-handed coordinate system. The center of the screen is at `X=0` and
+    /// `Y=0`.
+    ///
+    /// The default scaling mode is [`ScalingMode::WindowSize`], resulting in a resolution
+    /// where 1 unit in X and Y in camera space corresponds to 1 logical pixel on the screen.
+    /// That is, for a screen of 1920 pixels in width, the X coordinates visible on screen go
+    /// from `X=-960` to `X=+960` in world space, left to right. This can be changed by changing
+    /// the [`OrthographicProjection::scaling_mode`] field.
+    ///
+    /// The camera is placed at `Z=+1000-0.1`, looking toward the world origin `(0,0,0)`.
+    /// Its orthographic projection extends from `0.0` to `1000.0` in camera view space,
+    /// corresponding to `Z=+999.9` (closest to camera) to `Z=-0.1` (furthest away from
+    /// camera) in world space.
     pub fn new_2d() -> Self {
```

Comment:
Since you are adding docs for `new_2d`, you should probably also add some similar docs for `new_3d` below? It feels weird to leave it without docs.

The 3D ortho projection has the same right-handed coordinate system, but the default scaling mode is FixedVertical, spanning -/+ `-scale` along the vertical axis and -/+ `scale * aspect_ratio` along the horizontal. There is also no Z positioning weirdness.

Reply:
I have another PR ready which builds over this and adds docs to the entire `bevy_render::camera` module subtree, but touches the same files, so I'm waiting for this to merge first.